class.jetpack-gutenberg.php

```diff
@@ -947,7 +947,10 @@ public static function validate_block_embed_url( $url, $allowed = array(), $is_r
 	 * @return string
 	 */
 	public static function upgrade_nudge( $plan ) {
-		if ( ! current_user_can( 'manage_options' ) ) {
+		/** This filter is documented in _inc/lib/admin-pages/class.jetpack-react-page.php */
+		if ( ! apply_filters( 'jetpack_block_editor_enable_upgrade_nudge', false )
+			|| ! apply_filters( 'jetpack_show_promotions', true )
+			|| is_feed() ) {
```

Comment:
Do we need to leave `manage_options` too, so that this only shows for the admin of the site?

Reply:
Yes! My mistake, will add this back in.

Reply:
✅